src/gurt/fault_inject.c

```diff
@@ -180,6 +180,7 @@ fault_attr_set(uint32_t fault_id, struct d_fault_attr_t fa_in, bool take_lock)
 	fault_attr->fa_max_faults = fa_in.fa_max_faults;
 	fault_attr->fa_err_code = fa_in.fa_err_code;
 	fault_attr->fa_argument = fa_argument;
+	fault_attr->fa_num_faults = fa_in.fa_num_faults;
```

Comment:
could you add a  comment for d_fault_attr_set() in fault_inject.h to reflect this new change?

Reply:
ok